wiki/People/Beatmap_Nominators/id.md

```diff
@@ -0,0 +1,172 @@
+[false]: /wiki/shared/false.png "No"
+[true]: /wiki/shared/true.png "Yes"
+
+Beatmap Nominator
+==================
+
+*Lihat juga: [Peraturan Beatmap Nominator](/wiki/Beatmap_Nominator_Rules) dan [Informasi Umum untuk Beatmap Nominator](/wiki/Beatmap_Nominator_General_Information).*
+
+Beatmap Nominator, dikenal juga dengan *BN*, adalah kelompok yang bertanggung jawab atas [penominasian beatmap](/wiki/Submissions/Beatmap_Ranking_Procedure). Sejak 2014-08-22, Beatmap Nominator telah dipisahkan dari bagian staff, perubahan ini meliputi pencabutan hak moderasi di chat in-game, nama merah di in-game, dan di dalam forum. Pada 2015-02-01, nama kelompok berubah dari *Beatmap Appreciation Team* menjadi *Beatmap Nominators*. Saat ini, mereka memiliki warna ungu pada nama mereka di forum.
+
+Peran dan Tanggung Jawab
+--------------------------
+
+Anggota Beatmap Nominator bertanggung jawab terutama untuk [modding](/wiki/Modding) [beatmaps berstatus pending](/wiki/Beatmaps) ke dalam ranah status ranking. Beatmap Nominator adalah orang-orang yang dapat dimintai saran tentang mapping atau modding.
+
+Menjadi seorang Beatmap Nominator
+----------------------------
+
+Dengan perombakan team pada Agustus 2014 lalu, bergabung menjadi Beatmap Nominator jauh lebih mudah dibandingkan dengan yang dulu. [Quality Assurance Team](https://osu.ppy.sh/g/7) akan mengumumkan pendaftaran Beatmap Nominator berupa post di [Pending Beatmaps Subforum](https://osu.ppy.sh/forum/6) dalam kurun waktu 1 atau 2 bulan.
+
+### Kriteria Dasar
+
+Modder yang ingin menjadi Beatmap Nominator harus memenuhi kriteria berikut:
++ Harus aktif dalam modding setidaknya kurang lebih 3 bulan jauh dari pengumuman.
++ Mempunyai kurang lebih 50 [Kudosu](/wiki/Glossary/#kudosu) dari total yang sudah diakumulasi.
++ Memberikan hasil modding setidaknya 2 atau 4 mod agar memenuhi syarat untuk menjadi bagian dari Beatmap Nominator.
+
+### Apa yang Berkualitas sebagai Mod Post
+
++ Setelah mendaftar, Anda akan diminta untuk memberikan jumlah total Kudosu serta jumlah postingan mod dalam kurun waktu 3 bulan terakhir, diurutkan berdasarkan bulan setelah mendaftar. Jadi jika pengumuman diposting pada bulan April, Anda akan melampirkan posting mod bulan Januari, Februari, dan Maret akan relevan. Bulan-bulan yang relevan juga akan terdaftar pada formulir submission.
++ Sebuah mod post akan memenuhi syarat apabila sudah menerima Kudosu dan dibuat dalam rentang waktu yang sudah tercantum dalam formulir submissions.
++ Sejak November 2017, modding beatmap telah dirubah menggunakan Modding V2 yang memiliki sistem reward berdasarkan upvote/downvote. Oleh karena itu nilai "Mod Post" sangat diperlukan dalam diskusi beatmap, yang dimana setidaknya satu masalah/saran mendapatkan reward upvote.
```

Comment:
>Oleh karena itu nilai "Mod Post" sangat diperlukan dalam diskusi beatmap, yang dimana setidaknya satu masalah/saran mendapatkan reward upvote

The first half of this sentence is incorrectly translated, as it's more about explaining that a beatmap discussion _is the equivalent of_ "mod posts", not that a "mod post" _is necessary in_ the beatmap discussion itself, which is the current translation. `yang dimana` is also a phrase necessary to be pointed out, because `yang` is redundant in this sentence, and `dimana` should be separated and changed as `di mana`.

I suggest using something along the lines of `"Mod post" di sini sama dengan diskusi beatmap, di mana setidaknya satu saran atau masalah di dalamnya mendapatkan sebuah upvote.`